src/main/resources/templates/monthlyStats.html

```diff
@@ -51,7 +51,7 @@ <h2>Issues created per month in QuarkusIO GitHub organisation</h2>
                     {#for entry in stats.quarkusioPRs}
                     {
 						label: '{entry.key}',
```

Comment:
Why all these changes?

"index" no longer working in Qute ?

Reply:
See https://quarkus.io/guides/qute-reference#loop_section

Reply:
If it's not working issue needs to be filed

Reply:
looks that we were missing a reference to the parent object